Apps/WebClient/src/ClientApp/test/components/monthYearPicker.test.ts

```diff
@@ -1,22 +1,22 @@
 import "@/plugins/inversify.config";
-import { createLocalVue, shallowMount } from "@vue/test-utils";
 
+import { StoreOptionsStub } from "@test/stubs/store/store";
```

Comment:
"@test/stubs/store/store"; looks redundant to me

Reply:
Agreedd. I 've changed it to /otions